### PR TITLE
[Gamepad] Update - Add a function to check Xbox controllers

### PR DESCRIPTION
**Pull request by Bouh**
Check for "Xbox" and "Xinput" for Xbox Microsoft Controllers to fix a possible issue on Firefox that only detect "Xinput".

Function condition: **C_Controller_type** changed
And the function `isXbox()` used in the condition is in: **FirstSceneLoaded**

cc @Silver-Streak
cc @ClementPasteau

### CODE REVIEW

File: Extensions/Gamepads.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "name": "Gamepads",
   "previewIconUrl": "https://resources.gdevelop-app.com/assets/Icons/gamepad-variant-outline.svg",
   "shortDescription": "Add support for gamepads (or other controllers) to your game, giving access to information such as button presses, axis positions, trigger pressure, etc...",
-  "version": "0.0.10",
+  "version": "0.0.2",
```
Comment (4ian) [inline Extensions/Gamepads.json:12]: Should be increasing, either 0.2.0 or 0.0.11 :)

Comment (Bouh) [inline Extensions/Gamepads.json:12]:
```suggestion
  "version": "0.2.0",
```

**Comment by Bouh** [comment]
[This could help?](https://gist.github.com/Enichan/de1618d2c481523bd0c1123800bfe747)

**Comment by ClementPasteau** [comment]
That's a really interesting link, however it shows that it's almost impossible to know which gamepad is connected because of the browser 😓  (Opera and Safari are even worse lol)

**Comment by Bouh** [comment]
Ok then as we can't trust Xinput, I comment on the line below, and I merge this PR.
image

**Comment by 4ian** [comment]
Ok! :)

**Pull request merged**